### ISSUE TITLE
Generate IDEX total dust counts vs orbit position quicklook plots

### ISSUE THREAD

**Issue by bourque** [IMAP-Science-Operations-Center/imap_processing#837]
Will update this once we get a description/requirements for this

**Comment by tech3371** [comment]
I am un-assigning Luisa from the ticket at the moment. It's ok if someone is helping but nice to have one person who owns the ticket.

**Comment by mmshaw** [comment]
This issue is blocked by another issue (still to be opened):

- [@aldo9253] We are developing an algorithm to automatically distinguish dust events from noise events. We'll be using this algorithm to construct a new corresponding flag in our l1b and l2 data products. This will enable us to begin using level 3 data products: density maps. I can write this into a ticket.
